docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  app:
+    image: node:alpine
```

Comment:
Let's use `node:lts-alpine` for this instead (should work the same).

Reply:
No, it should know about Tom. I'll take a look